content/events/2018/competition.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Competition 2018
+date: 2018-04-21 09:00:00
+end_date: 2018-04-22 17:00:00
+all_day: false
+---
+
+## Schedule
+The competition will run over 2 days, starting on Saturday morning, and finishing on Sunday afternoon. Exact times to be confirmed.
+
+## Media Consent
+Media consent will be required to attend the competition. These will need to be signed by each member of the team, as well as staff. Those under 18 will need a parent / guardian to sign on their behalf.
```

Comment:
Perhaps explain what media consent is and why we need it?

Reply:
After racking my brain, I'm not exactly sure how to describe what it's for? Besides allowing us to use photos of the events for various things, which can contain their likeness.

Reply:
How about this:
> In order to take photos and videos for promotional use (mainly on our website), we are required to collect a media consent release. We therefore require everyone (including team leaders, competitors and guests) who attends the the competition to complete a media consent form. Under 18s need a parent or guardian to sign on their behalf.